addon-test-support/-private/helpers.js

```diff
@@ -38,7 +38,7 @@ class Selector {
     }
 
     deprecate(
-      'Usage of comma separated selectors is deprecated in ember-cli-page-object', selector.indexOf(',') === -1, {
+      'Usage of comma separated selectors is deprecated in ember-cli-page-object', !this.isCommaSeparated(selector), {
```

Comment:
I think the problem here is that we just do the check too late, when the `targetFilters` are applied already.
Can we move the check somewhere before it happened https://github.com/san650/ember-cli-page-object/pull/521/files#diff-7f9866fcc737dac901e9ba9b15adae0045477e229889aaf75ec3f343550d45f0R32, in order to avoid working around the filters?